### PR TITLE
Allow collect only without running cluster

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes issue #628
This allow us to run collect-only option with pytest to check some import errors and collection issues.

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
I created one testing branch to test some import issue here: 
https://github.com/petr-balogh/ocs-ci/commit/a89d79062c51bfa981dbb03b82683b46aa375334
Results which are failing our CI:
https://travis-ci.org/petr-balogh/ocs-ci/builds/575738884

File: ocs_ci/framework/pytest_customization/reports.py [hunk 1]
```diff
@@ -45,5 +45,5 @@ def pytest_sessionfinish(session, exitstatus):
     """
     send email report
     """
-    if ocsci_config.RUN['cli_params']['email']:
+    if ocsci_config.RUN['cli_params'].get('email'):
```
Comment (vasukulkarni) [inline ocs_ci/framework/pytest_customization/reports.py]: This seem like "add-on" commit,  please remove this line.

Comment (petr-balogh) [reply ocs_ci/framework/pytest_customization/reports.py]: This is required otherwise failing with keyerror

Comment (vasukulkarni) [reply ocs_ci/framework/pytest_customization/reports.py]: weird but we should revisit this one to find out more

**Pull request merged**